app/assets/stylesheets/finder_frontend.scss

```diff
@@ -370,3 +370,12 @@ mark {
     }
   }
 }
+
+// remove separator and increase spacing between list items
+// later, propose both features as flags in the 
+// "document list" govuk-publishing component
+.gem-c-document-list__item {
+  border: 0;
+  margin-bottom: govuk-spacing(7);
```

Comment:
It looks like the margin bottom could be a little smaller, so we could get more results into the screen at the same time. Is there a reason `govuk-spacing(7);` was chosen?

Reply:
design by JeremyTM

Reply:
In that case :shipit: 🚢 ⚓️